### ISSUE TITLE
Missing tooltips for icons in mail viewer header

### ISSUE THREAD

**Issue by juni2k** [tutao/tutanota#7586]
- [X] I've searched and did not find a similar issue.

**Expected behaviour:** tooltips pop out when hovering over mail icons, providing some explanation
**Reality:** it seems that only the folder icon displays a tooltip

![Hovering over folder icon (Inbox), a tooltip is displayed](https://github.com/user-attachments/assets/dec49496-f731-49ae-8133-3b9912b89003)
![Hovering over the "confidential" icon, no tooltip is displayed](https://github.com/user-attachments/assets/a10861df-3e29-425f-9311-d909b85e93d2)

**Comment by juni2k** [comment]
This issue was motivated by me not knowing what the PQ-confidential icon was meant to represent. Due to how the code is structured, both the PQ-confidential and the regular confidential icon are now getting a "Confidential" tooltip which is a bit better than nothing, but still doesn't clear up the difference. Then again, I don't know whether such a technical difference should be exposed to end users in the first place. Need more feedback.
